Sample.BasicComConnector/NewSimClient.cs

```diff
@@ -28,6 +29,12 @@ public NewSimClient(ILogger<NewSimClient> logger, DefaultConfig<NewSimAutomation
         }
     }
 
+    public Task TestConnection(CancellationToken _token)
+    {
+        /* Attempt to connect to the simulator */
+        return Task.CompletedTask;
+    }
```

Comment:
```suggestion
    public Task TestConnection(CancellationToken _token)
    {
        return Task.CompletedTask;
    }
```
this comment doesn't make sense here